src/promptflow/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 - Display node status in run visualize page graph view.
 - Add support for image input and output in prompt flow.
 - [SDK/CLI] SDK/CLI will collect telemetry by default, user can use `pf config set telemetry.enabled=false` to opt out.
+- Enable specifying the default output path for run.
+  - Use `pf config set run.output_path=<output-path>` to specify, and the run output path will be `<output-path>/.runs/<run-name>`.
```

Comment:
shall user point to <output-path>/.runs?

Reply:
${flow_directory}/.runs or ${flow_directory}/.promptflow/.runs

Reply:
current I will simply append a `/.runs`, so if user specify `<somewhere>/.runs`, then the final place will be `<somewhere>/.runs/.runs`

Reply:
do you think we should append `/.promptflow/.runs`?

Reply:
I mean we should not append anything. User should append himself if he wants the above path.

run.output_path: ${flow_directory}/.runs

Reply:
as offline synced, not append anything, but ban some pattern and check for it.